hazelcast/src/main/java/com/hazelcast/logging/LoggingService.java

```diff
@@ -25,11 +25,36 @@
 
     void removeLogListener(@Nonnull LogListener logListener);
 
+    /**
+     * Creates a logger with the given name.
+     * <p>
+     * The returned logger must be released using {@link #removeLogger(String)}
+     * when no longer used.
+     *
+     * @param name the logger name
+     * @return the logger with the desired name
+     */
     @Nonnull ILogger getLogger(@Nonnull String name);
 
+    /**
+     * Creates a logger using the class name of the given class as the logger
+     * name.
+     * <p>
+     * The returned logger must be released using {@link #removeLogger(Class)}
+     * when no longer used.
+     *
+     * @param type the class whose name use as the logger name
+     * @return the logger with the desired name
+     */
     @Nonnull ILogger getLogger(@Nonnull Class type);
 
+    /**
+     * Release resources associated with the given logger.
+     */
     void removeLogger(@Nonnull String name);
```

Comment:
nit: would be nice to document the param here.

Reply:
To me it's just a noise. The single sentence describes the parameter.

Reply:
But you documented them in other methods, so it's a matter of consistency.

Reply:
We don't require such consistency. Docs should be consistently concise and precise. Adding `@param name the logger name` doesn't give much additional information.

Reply:
It's up to you to decide. That's why it's a nit.